src/styles/scenes/SettingsStyle.js

```diff
@@ -69,6 +69,11 @@ export const styles = {
     backgroundColor: THEME.COLORS.TRANSPARENT
   },
 
+  // /////// Double Row
+  doubleRowContainer: {
+    height: scale(88),
+    backgroundColor: THEME.COLORS.WHITE
```

Comment:
Do we really need to set a height for the container? In general I think it's better to not hard-code heights if we don't have to.

Reply:
ok, removed. I was just looking at what we did for single row. since the rows are defined and we are having two in the container, we can remove